tests/console/snapper_create.pm

```diff
@@ -19,7 +19,17 @@ sub get_last_snap_number {
     # In many cases script output returns not only script execution results
     # but other data which was written to serial device. We have to ensure
     # that we got what we expect. See poo#25716
-    my $output = script_output("snapper list | tail -n1 | awk '{ print \">>>\" \$3 \"<<<\" }'");
+
+    # get snapshot id column
+    my $snap_head         = script_output("snapper list | head -n1");
+    my $snap_id_col_index = 1;
+    for my $field (split(/\|/, $snap_head)) {
+        $field =~ s/^\s+|\s+$//g;    # trim spaces
+        last if ($field eq '#');     # get snapshot id field
+        $snap_id_col_index++;
+    }
+
```

Comment:
Just one small missing part, we need to give a hint if parsing fails, so if no column was identified, we need to die with clear message, otherwise one can spend quite some time for investigation.